<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportInfo</class>
 <widget class="QWidget" name="ReportInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextBrowser" name="ReportDisplay">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>40</y>
     <width>581</width>
     <height>391</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="LogOutButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Log Out</string>
   </property>
  </widget>
  <widget class="QPushButton" name="testButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>200</y>
     <width>161</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>test std::string to GUI</string>
   </property>
  </widget>
  <widget class="QPushButton" name="testItemButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>310</y>
     <width>171</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>test specific item report
(&quot;Shrek 2&quot;)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="testDayButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>410</y>
     <width>171</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>test specific day report 
(&quot;day1.txt&quot;)</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="DayBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>450</y>
     <width>48</width>
     <height>29</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>5</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>460</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Days 1-5:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="ItemSearchInput">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>360</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>319</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Search Item</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
